### ISSUE TITLE
Not run on GPU although using full of GPU memory

### ISSUE THREAD

**Issue by xujingao13** [matterport/Mask_RCNN#595]
I'm using the package tensorflow-gpu==1.5.0, installed from `conda`. When I run the samples/demo.py, I found although the memory-usage of the GPU is nearly 90%, the GPU-Util is still 0% and the program is running so slow..... How can I do about this? Thanks a lot!

There are some screenshots:
- The logs of the script:
![info](https://user-images.githubusercontent.com/8785750/40573202-ee8b613a-60ef-11e8-8cb4-9308924382ae.png)

- The memory-usage and GPU-Util of my GPU
![gpu](https://user-images.githubusercontent.com/8785750/40573208-fc026cd2-60ef-11e8-85a9-e7c8bba8c860.png)

- The usage of CPU
![cpu](https://user-images.githubusercontent.com/8785750/40573210-00fc53e2-60f0-11e8-9a51-e95286310520.png)

**Comment by zungam** [comment]
You are on linux? You run NVIDIA-SMI? Sometimes that function doesnt update real time. If your load_image and load_mask functions are slow, the GPU halts to wait for CPU to finish. If you do nvidia-smi at that exact instance, the GPU util will be 0%. Do

`watch -n0.1 nvidia-smi`

for fast update to monitoring it in real time. If it is utilized some, but not all the time, it means your load_mask or load_image function is slow.

**Comment by leaderkent** [comment]
I have the same problem, have you solved it?

**Comment by HanYuanyuaner** [comment]
@zungam So how can I fix this problem?

**Comment by zungam** [comment]
Everyone can confirm that the GPU is never utilized?

**Comment by HanYuanyuaner** [comment]
Yeah, I used to your approach to check it, the GPU is never utilized. Do you know the reason?

**Comment by zgxsin** [comment]
@zungam I also have this problem. As far as I know, when we finish loading images, the model should start training. And we will never need to load the images again. Am I right? If so, then the GPU usage should be normal after this stage.

**Comment by jyning12** [comment]
Same problem...

**Comment by mzlt** [comment]
same here, anybody got a fix for this one?

**Comment by yiyang186** [comment]
the same problem ... after running some epochs

**Comment by saskra** [comment]
Same here.

**Comment by ceciliaAI** [comment]
same problem, so anyone who solved this problem?
